code/controllers/subsystem/abnormality_queue.dm

```diff
@@ -29,13 +30,10 @@ SUBSYSTEM_DEF(abnormality_queue)
 	var/hardcore_roll_enabled = FALSE
 
 /datum/controller/subsystem/abnormality_queue/Initialize(timeofday)
-	var/list/all_abnos = subtypesof(/mob/living/simple_animal/hostile/abnormality)
-	for(var/i in all_abnos)
-		var/mob/living/simple_animal/hostile/abnormality/abno = i
-		if(initial(abno.can_spawn))
-			possible_abnormalities[initial(abno.threat_level)] += abno
 	if(LAZYLEN(possible_abnormalities))
 		pick_abno()
+	else
+		stack_trace("[src] initialized before Persistence subsystem!")
```

Comment:
As per code standards.
```suggestion
	if(!LAZYLEN(possible_abnormalities))
		stack_trace("[src] initialized before Persistence subsystem!")
		return
	pick_abno()
```

Reply:
Doing this prevents starting room count from being set, next abno spawn time from being decremented, and parent call from being sent. This is intentionally set this way as if Persistence loads later, the initial spawn will still work it'll just not give the manager that first choice.

Reply:
If that's the case, it should be fine.